### ISSUE TITLE
Implement a separate changelog

### ISSUE THREAD

**Issue by hskrasek** [Bungie-net/api#342]
@vthornheart-bng This is more of a question for you, but would you be willing to implement a separate CHANGELOG.md? Maybe something that follows along with the format [here](http://keepachangelog.com/en/1.0.0/)?

Copy-pasted format blurb for anyone that wants to see the format but not leave the issue

```markdown
# Changelog
All notable changes to this project will be documented in this file.

The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).

## [Unreleased]

## [1.0.0] - 2017-06-20
### Added
- New visual identity by @tylerfortune8.
- Version navigation.
- Links to latest released version in previous versions.
- "Why keep a changelog?" section.
- "Who needs a changelog?" section.
- "How do I make a changelog?" section.
- "Frequently Asked Questions" section.
- New "Guiding Principles" sub-section to "How do I make a changelog?".
- Simplified and Traditional Chinese translations from @tianshuo.
- German translation from @mpbzh & @Art4.
- Italian translation from @azkidenz.
- Swedish translation from @magol.
- Turkish translation from @karalamalar.
- French translation from @zapashcanon.
- Brazilian Portugese translation from @aisamu.
- Polish translation from @amielucha.
- Russian translation from @aishek.
- Czech translation from @h4vry.
- Slovak translation from @jkostolansky.
```

**Comment by vthornheart-bng** [comment]
Ah, yes!  Can do - I'll get that added soon.

**Comment by hskrasek** [comment]
@vthornheart-bng Woot! If I can help in anyway, wouldn't mind submitting a PR

**Comment by vthornheart-bng** [comment]
Oh, no worries - it'll be easy enough for me to do once I get in tomorrow (er, today, just turned midnight), I just was preoccupied with other activities today!

**Comment by vthornheart-bng** [comment]
There we go.  I'm not going to bother trying to dig up all of the info for what we released between the initial D2 launch and when I started getting my act together and putting out real changelog info earlier this month... it'd take a while to dig up all of that information, and I don't know how useful it would be at this point.  But starting with 2.1.0 and from here on out I will strive to make these complete and informative!
